src/menu/views/settings_editor.c

```diff
@@ -26,44 +121,51 @@ static void draw (menu_t *menu, surface_t *d) {
 
 	component_main_text_draw(
         ALIGN_CENTER, VALIGN_TOP,
-        "SETTINGS EDITOR\n"
+        "MENU SETTINGS EDITOR\n"
         "\n"
     );
 
     component_main_text_draw(
         ALIGN_LEFT, VALIGN_TOP,
-        "\n"
-        "\n"
-        "To change the settings, please adjust them\n"
-        "directly in the 'menu/config.ini' file.\n\n"
-        "pal60_enabled:          %s\n"
-        "show_protected_entries: %s\n"
-        "default_directory:      %s\n"
-        "use_saves_folder:       %s\n"
-        "bgm_enabled:            %s\n"
-        "sound_enabled:          %s\n"
-        "rumble_enabled:         %s\n",
+        "\n\n"
+        "To change the menu settings, press 'A'.\n\n"
+        "*    Default Directory : %s\n"
+        "     PAL60 Mode        : %s\n"
+        "     Show Hidden Files : %s\n"
+        "     Use Saves folder  : %s\n"
+        "**   Sound Effects     : %s\n"
+        "***  Background Music  : %s\n"
+        "***  Rumble Feedback   : %s\n\n\n"
```

Comment:
do we _really_ need these settings exposed even if these does absolutely nothing? I'm not a fan of broken functionality, like this or empty screen, exposed to the end user

Reply:
I guess not, but, they are in the settings... For the develop branch it would be fine (as it allows them to be implemented)
Suggestions would be to 
* just keep for the moment
* wrap in an ifdef
* remove entirely

Reply:
it's best to keep these unfinished features behind ifdef, you can have build time define that enables all features still in development while main branch have those disabled by default

Reply:
Done.